apps/meteor/tests/end-to-end/api/livechat/11-livechat.ts

```diff
@@ -102,6 +105,66 @@ describe('LIVECHAT - Utils', function () {
 
 			await deleteCustomField(customFieldName);
 		});
+		(IS_EE ? it : it.skip)('should return online as true if there is at least one agent online', async () => {
```

Comment:
am I the only one who doesn't like this? why not using like this?

```suggestion
		it('should return online as true if there is at least one agent online', async () => {
			if (!IS_EE) {
				this.skip();
			}
```

Reply:
https://www.reddit.com/r/ProgrammerHumor/comments/b6z1fc/ternaries_are_very_sophisticated/

Reply:
PS: I usually go with this approach because it seems more compact, but if you want me to change it, let me know :)